package.json

```diff
@@ -9,10 +9,17 @@
     "owner": "Enplug",
     "name": "dashboard-sdk-utils"
   },
+  "scripts": {
+    "i18n": "npm run i18n:extract && npm run i18n:compile",
```

Comment:
See https://sites.google.com/enplug.com/home/development/translations for how latest version of scripts look (and install necessary dependencies).

It changed a bit because on Windows machines, appropriate directories weren't created and scripts threw errors.